Makefile

```diff
@@ -377,7 +379,6 @@ az-patch-install-ragengine-helm-e2e:
 aws-patch-install-helm:
 	yq -i '(.image.repository)                                              = "$(REGISTRY)/workspace"'                    	./charts/kaito/workspace/values.yaml
 	yq -i '(.image.tag)                                                     = "$(IMG_TAG)"'                               	./charts/kaito/workspace/values.yaml
-	yq -i '(.featureGates.Karpenter)                                    	= "true"'                                       ./charts/kaito/workspace/values.yaml
```

Comment:
fyi - I went through this @Jont828, this feature gate does not exist. that's why we are removing it